imagersite/imagersite/production.py

```diff
@@ -0,0 +1,8 @@
+from settings import *
+
+DEBUG = False
+TEMPLATE_DEBUG = False
+ALLOWED_HOSTS = ['ec2-52-26-163-206.us-west-2.compute.amazonaws.com',
```

Comment:
This is potentially a transient value.  Every  time you set up a new server, the public DNS for that server will be different.  And if you shut down and then restart an existing EC2 instance, the public DNS will also change.  Don't write this type of data to your settings file.  Get it from your environment instead, if you can.